l10n_it_fatturapa/bindings/donotapply.fatturapa.diff

```diff
@@ -0,0 +1,25 @@
+diff --git a/l10n_it_fatturapa/bindings/fatturapa.py b/l10n_it_fatturapa/bindings/fatturapa.py
```

Comment:
Questo file quindi lo eliminiamo?

Reply:
fatto su #1900 (il binding.diff era quello in uso, non l'ho ri-verificato)